### PR TITLE
Adds lacework tile for lacework integration

### PR DESCRIPTION
**Pull request by anshumgargdd**
### What does this PR do?

This PR adds basic scaffolding and tile for Lacework integration

### Motivation

We recently released a Lacework logs integration which needs an in app tile.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Comment by hithwen** [comment]
We are currently on a code freeze, let's wait till next week to merge this

File: lacework/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Agent Check: Lacework
```
Comment (l0k0ms) [inline lacework/README.md]: If we intend to store the integrations content here, let's be consistent with the current lacework integration documentation already in the doc: https://docs.datadoghq.com/integrations/lacework/

Source file: https://github.com/DataDog/documentation/blob/master/content/en/integrations/lacework.md

**Comment by anshumgargdd** [comment]
@hithwen Can we merge this now?

**Comment by hithwen** [comment]
> @hithwen Can we merge this now?

Yes, but have you addressed @l0k0ms's feedback?

**Comment by anshumgargdd** [comment]
@hithwen Rebased and Updated readme based on feedback. Free to merge. Thanks!

**Comment by anshumgargdd** [comment]
Hello @hithwen rebumping. Can we merge this?

File: lacework/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Overview
+
+Collect Lacework Logs or Events.
```
Comment (l0k0ms) [inline lacework/README.md]:
```suggestion
Use the Datadog-Lacework integration to forward your Lacework logs and events to Datadog.
```

File: lacework/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+## Setup
+
+All configuration happens on Lacework Dashboard. Find details on how to set it up in [Datadog documentation][1] or [Lacework Documentation][2]
```
Comment (l0k0ms) [inline lacework/README.md]: This file acts as the Datadog documentation, no need to reference it.

```suggestion
All configuration happens on Lacework Dashboard. Find details on how to set it up in the [Lacework Documentation][2]. Datadog automatically enables the right logs processing pipeline when it detects Lacework logs.
```

**Comment by anshumgargdd** [comment]
@l0k0ms Accepted suggestions. Feel free to merge. Thanks.

**Comment by anshumgargdd** [comment]
@hithwen Good to merge? Thanks!

**Pull request merged**